views/default/html_email_handler/notification/body.php

```diff
@@ -41,7 +41,7 @@
 	$notification_footer = elgg_echo('html_email_handler:notification:footer:settings', [
 		"<a href='{$settings_url}'>",
 		'</a>',
-	]);
+	], $recipient->language);
```

Comment:
I believe you can use `$language` which is defined at the start of the file and is the same language as available in the notification hooks (which is the recipient language)

Reply:
In most of the cases, the recipient user is the logged in user. But, in some cases as #43, the recipient user language is not the same language as the logged in user language.

Reply:
In most cases the **sender** is the logged in user, the recipient is somebody else.

I checked i'm not passing language yet. And i think `$language` should instead be defaulted to `$recipient->getLanguage()`

Reply:
I agree